src/cloud/project/magento-app-php-application.md

```diff
@@ -41,96 +41,19 @@ php -m
 
 For details about a specific PHP extension, see the [PHP Extension List](https://www.php.net/manual/en/extensions.alphabetical.php).
 
-{{site.data.var.ece}} supports the following extensions:
+The following table shows the supported PHP extensions when deploying {{site.data.var.ee}} on the Cloud platform.
 
--  Default extensions:
-   -  `bcmath`
-   -  `bz2`
-   -  `calendar`
-   -  `exif`
-   -  `gd`
-   -  `gettext`
-   -  `intl`
-   -  `mysqli`
-   -  `pcntl`
-   -  `pdo_mysql`
-   -  `soap`
-   -  `sockets`
-   -  `sysvmsg`
-   -  `sysvsem`
-   -  `sysvshm`
-   -  `opcache`
-   -  `zip`
+-  **Required extensions** install automatically and cannot be removed.
```

Comment:
It's confusing because the runtime section has two sections:  extensions:  and disabled:extensions
```
    extensions:
        - sockets
        - sodium
        - ssh2
    disabled_extensions:
        - bcmath
        - bz2
        - calendar
        - exif
```
and the headings in the original topic for each section were:

- default extensions
- Extensions that are installed and cannot be uninstalled
- Extensions that can be installed and uninstalled as needed

It's not clear to me what is meant by install and uninstalled --  and how that relates to the runtime section.